qiime2/core/testing/plugin.py

```diff
@@ -160,9 +160,9 @@
         'int1': Int,
         'int2': Int
     },
-    outputs=[
-        ('concatenated_ints', IntSequence1)
-    ],
+    outputs={
+        'concatenated_ints': IntSequence1
+    },
```

Comment:
These are only a few, but commonly used dummy plugin actions to demonstrate that this works. There isn't a corresponding `test_signature.py` and most of that setup code is exercised heavily in other tests (such as these).